accepted/2021/package-validation/package-validation.md

```diff
@@ -0,0 +1,236 @@
+# Package Validation
```

Comment:
On my first read, I confused "Package Validation" with [nuget package verification](https://github.com/dotnet/announcements/issues/180).

Reply:
Perhaps another name makes sense here since many users confuse the verification feature with validation already.